runtime/plugin/tutor-mode.vim

```diff
@@ -0,0 +1,12 @@
+if exists('g:loaded_tutor_mode_plugin') || &compatible
```

Comment:
Isn't it conventional to use the same name for `plugin/`, `autoload/`, `syntax/`, etc., typically?